### PR TITLE
Adding table retention days property

### PR DESCRIPTION
**Pull request by 123subhadeep**
Tested doing "DESC FORMATTED" on the created table in hive.

### CODE REVIEW

**Comment by 123subhadeep** [comment]
Will fix the build

**Comment by electrum** [comment]
This commit message is wrong.

File: presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java [hunk 1]
```diff
@@ -144,6 +144,7 @@
     private final TypeManager typeManager;
     private final JsonCodec<PartitionUpdate> partitionUpdateCodec;
     private final ExecutorService renameExecutionService;
+    private TablePropertiesSetter tablePropertiesSetter;
```
Comment (electrum) [inline presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java]: This should be final. Inject the property rather than using a setter.

**Comment by 123subhadeep** [comment]
I have updated the commit message and applied review comments.

**Comment by 123subhadeep** [comment]
I have added the retention property to HiveTableProperties.

**Comment by 123subhadeep** [comment]
Will create a new pull request.

**Pull request closed without merging**